### PR TITLE
Tutorial workbench

### PR DESCRIPTION
**Pull request by xi**
This is a sample workbench implementation for a workshop. **Do not merge!** It will be converted to a proper tutorial later.

### CODE REVIEW

File: src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/MyPackage/MyProposal/Detail.html [hunk 1]
```diff
@@ -1,4 +1,5 @@
 <article class="my-proposal">
     <h2>{{ data.title }}</h2>
     <p>{{ data.description }}</p>
+    <a data-ng-href="{{ path | adhResourceUrl:'comments' }}">comments</a>
```
Comment (xi) [inline src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/MyPackage/MyProposal/Detail.html]: Should be item url, not version url!

File: src/mercator/mercator/static/js/Adhocracy.ts [hunk 2]
```diff
@@ -178,6 +180,7 @@ export var init = (config : AdhConfig.IService, metaApi) => {
     AdhMarkdownModule.register(angular);
     AdhMercatorModule.register(angular);
     AdhMovingColumnsModule.register(angular);
+    AdhMyPackage.register(angular);
     AdhPermissionsModule.register(angular);
     AdhPreliminaryNamesModule.register(angular);
     AdhProcessModule.register(angular);
```
Comment (xi) [inline src/mercator/mercator/static/js/Adhocracy.ts]: not supposed to be here

File: src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/MyPackage/MyProposal/Module.ts [hunk 3]
```diff
@@ -8,7 +9,11 @@ export var moduleName = "adhMyPackageMyProposal";
 export var register = (angular) => {
     angular
         .module(moduleName, [
+            AdhEmbedModule.moduleName,
             AdhHttpModule.moduleName,
         ])
+        .config(["adhEmbedProvider", (adhEmbedProvider : AdhEmbed.Provider) => {
+            adhEmbedProvider.registerDirective("my-proposal-detail");
```
Comment (der-john) [inline src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/MyPackage/MyProposal/Module.ts]: why isn't the name `adh-my-proposal-detail`?

Comment (xi) [inline src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/MyPackage/MyProposal/Module.ts]: That's just the way `adhEmbedProvider` works. I do not say that it is good though.

**Comment by xi** [comment]
This is no longer relevant

**Comment by der-john** [comment]
I disagree.

**Comment by der-john** [comment]
Until I do that, this is irrelevant, after all.

**Pull request closed without merging**